src/assisted_test_infra/download_logs/download_logs.py

```diff
@@ -200,7 +198,8 @@ def download_logs(
                     # Skip sleeping on last retry
                     if i < MAX_RETRIES - 1:
                         log.info(f"Going to retry in {retry_interval} seconds")
-                        time.sleep(retry_interval)
+
+                    [time.sleep(1) for _ in range(retry_interval)]  # allow Ctrl+C when running locally
```

Comment:
why do we need this?
if I'm doing ``python -c "import time; time.sleep(20)"``` I have no problem killing it

Reply:
My mistake, the KeyboardInterrupt does kill the sleep program (but I don't think that the waiting package)